src/main/java/org/prebid/server/bidder/openx/OpenxBidder.java

```diff
@@ -74,9 +79,25 @@ public Result<List<HttpRequest<BidRequest>>> makeHttpRequests(BidRequest bidRequ
     }
 
     @Override
+    public CompositeBidderResponse makeBidderResponse(BidderCall<BidRequest> httpCall, BidRequest bidRequest) {
+        try {
+            final OpenxBidResponse bidResponse = mapper.decodeValue(httpCall.getResponse().getBody(),
+                    OpenxBidResponse.class);
+            return CompositeBidderResponse.withBids(extractBids(bidRequest, bidResponse), extractFledge(bidResponse));
+        } catch (DecodeException e) {
+            return CompositeBidderResponse.withError(BidderError.badServerResponse(e.getMessage()));
+        }
+    }
+
+    /**
+     * @deprecated for this bidder in favor of @link{makeBidderResponse} which supports additional response data
+     */
+    @Override
+    @Deprecated(forRemoval = true)
```

Comment:
What about to just remove this method?

Reply:
Needed to satisfy the interface. I will empty it out and return an error instead.